core/src/main/java/com/epam/esm/dto/ErrorResponseDTO.java

```diff
@@ -1,10 +1,19 @@
 package com.epam.esm.dto;
 
-import org.springframework.http.HttpStatus;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
 
-public record ErrorResponseDTO(
-        String errorMessage,
-        int errorCode,
-        HttpStatus httpStatus
-) {
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Builder
+public class ErrorResponseDTO {
```

Comment:
It would be better if you write javadoc for DTOs, enums, and exceptions to describe the purpose of these classes.